tmt/steps/provision/bootc.py

```diff
@@ -152,6 +152,15 @@ class BootcData(tmt.steps.provision.testcloud.ProvisionTestcloudData):
              """,
     )
 
+    build_disk_image_only: bool = field(
+        default=False,
+        is_flag=True,
+        option=('--build-disk-image-only'),
+        help="""
+             Only build a bootc disk image from a container image.
```

Comment:
To me, this wasn't clear until I read the code, as it does not say the important part - only build an image, *and will not boot it up*. E.g. "Only build a bootc disk image from a container image and quit. Guest VM will not start." would be clearer to me.

Reply:
Updated.